tools/test-proxy/Azure.Sdk.Tools.TestProxy/Azure.Sdk.Tools.TestProxy.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+  <PropertyGroup>
+    <TargetFrameworks>net5.0</TargetFrameworks>
+    <RequiredTargetFrameworks>$(TargetFrameworks)</RequiredTargetFrameworks>
+    <NoWarn>$(NoWarn);1591;AZC0001;AZC0012;CA1724;CA1801;CA1812;CA1822;SA1028</NoWarn>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <IsPackable>true</IsPackable>
+    <PackAsTool>true</PackAsTool>
+    <ToolCommandName>test-proxy</ToolCommandName>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Azure.Core" Version="1.10.0" />
+    <PackageReference Include="LibGit2Sharp" Version="0.26.2" />
```

Comment:
Is the git lib dependency still needed?

Reply:
Given our discussions today...probably not. I'll clean it out.

Reply:
Done.